src/containers/ImportResources/ImportResources.js

```diff
@@ -225,47 +246,77 @@ export class ImportResources extends Component {
           Import Tekton resources from repository
         </h1>
         <Form>
-          <TextInput
-            data-testid="repository-url-field"
-            helperText="The location of the YAML definitions to be applied (Git URL's supported)"
-            id="import-repository-url"
-            invalid={this.state.invalidInput}
-            invalidText="Please submit a valid URL"
-            labelText="Repository URL"
-            name="repositoryURL"
-            onChange={this.handleTextInput}
-            placeholder="https://github.com/my-repository"
-            required
-            type="URL"
-            value={this.state.repositoryURL}
-          />
-          <NamespacesDropdown
-            id="import-namespaces-dropdown"
-            helperText="The namespace in which the resources will be created"
-            invalid={this.state.invalidNamespace}
-            invalidText="Please select a namespace"
-            onChange={this.handleNamespace}
-            required
-            selectedItem={selectedNamespace}
-          />
-          <TextInput
-            data-testid="directory-field"
-            helperText="The location of the Tekton resources to import from the repository. Leave blank if the resources are at the top-level directory."
-            id="import-directory"
-            labelText="Repository directory (optional)"
-            name="directory"
-            onChange={this.handleTextInput}
-            placeholder="Enter repository directory"
-            value={this.state.directory}
-          />
-          <ServiceAccountsDropdown
-            className="saDropdown"
-            helperText="The ServiceAccount that the PipelineRun applying resources will run under (must be in the install namespace of the Tekton Dashboard)"
-            id="import-service-accounts-dropdown"
-            namespace={this.state.installNamespace}
-            onChange={this.handleServiceAccount}
-            titleText="ServiceAccount (optional)"
-          />
+          <FormGroup legendText="Source repository and target namespace parameters">
```

Comment:
```suggestion
          <FormGroup legendText="Source repository and target namespace">
```

avoids confusion with `params` / `parameters` used elsewhere to refer to Pipeline / Task input params.